src/components/WwwFrame/TheFooter.vue

```diff
@@ -1,32 +1,36 @@
 <template>
-	<footer class="www-footer" :style="cssVars">
+	<footer class="tw-min-h-[17.5rem] lg:tw-min-h-[25rem] tw-p-2 md:tw-p-3
+				lg:tw-px-2 lg:tw-py-3 tw-bg-tertiary tw-text-primary print:tw-p-0 print:tw-font-book print:tw-h-auto">
 		<disclaimers />
-		<nav class="small-footer hide-for-large" aria-label="Footer navigation">
-			<div class="row collapse">
-				<div class="column">
-					<h2>Borrow</h2>
-					<p>Loans for entrepreneurs doing amazing things.</p>
-					<ul>
+		<nav class="tw-block lg:tw-hidden" aria-label="Footer navigation">
+			<div class="tw-mt-3 tw-mb-3">
+				<div>
```

Comment:
Are these wrapper divs needed anymore? If not, removing them can help with performance.

Reply:
Done!